src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/common.hpp

```diff
@@ -195,6 +199,64 @@ static constexpr size_t vec_len_f32_neon = vec_len_neon / sizeof(float);
     }
 #endif
 
+#ifdef OPENVINO_ARCH_ARM64
+    inline float32x4_t exp_ps_neon(float32x4_t& src) {
+        const auto c1 = vreinterpretq_f32_u32(vdupq_n_u32(0x3f7ffff6));
+        const auto c2 = vreinterpretq_f32_u32(vdupq_n_u32(0x3efffedb));
+        const auto c3 = vreinterpretq_f32_u32(vdupq_n_u32(0x3e2aaf33));
+        const auto c4 = vreinterpretq_f32_u32(vdupq_n_u32(0x3d2b9f17));
+        const auto c5 = vreinterpretq_f32_u32(vdupq_n_u32(0x3c072010));
+
+        const auto shift   = vreinterpretq_f32_u32(vdupq_n_u32(0x4b00007f)); // 2^23 + 127 = 0x1.0000fep23f
+        const auto one   = vdupq_n_f32(1.0f); // 1
+        const auto two   = vdupq_n_f32(2.0f); // 2
+        const auto inv_ln2 = vreinterpretq_f32_u32(vdupq_n_u32(0x3fb8aa3b)); // 1 / ln(2) = 0x1.715476p+0f
+        const auto neg_ln2_hi =
+                vreinterpretq_f32_u32(vdupq_n_u32(0xbf317200)); // -ln(2) from bits  -1 to -19: -0x1.62e400p-1f
```

Comment:
nit: 160 symbols is allowed in one line, not necessary to move on the next line.